### DISCUSSION TITLE
Min/Max Transition Curve?

### DISCUSSION THREAD

**Discussion by 325racer** [Rem0o/FanControl.Releases#1956] [General]
Is there a way to have a Min/Max transition curve? 

I have some auxiliary fans that I would like to turn off when my GPU drops below a certain temp, but I would also like to have them ramp up and run at the higher speed of either GPU or CPU above a certain point.

I previously tried a "Max Temp Mix" from my GPU and CPU curves, but my CPU never goes idle.

On the other hand I tried "Min Temp Mix", which will go idle, but then the MIN speed isn't always as high as I'd like.

I suppose I could just make a curve for these specific fans vs using a Mix.

Any other ideas?

**Comment by Rem0o** [comment]
Instead of mixing temp sensors, mix fan curves.  Create one for your GPU temp, one for your CPU temp, then Mix those 2 with a max function.   You will always get the right %/RPM depending if either your CPU or GPU gets hot.

**Reply by 325racer** [reply]
Sorry, for confusion, this is what I did, however, my GPU fans shut off below a certain point and I wanted these other fans to also shut off.

I guess what I want is to mix a Min and Max so that below a certain point it's either Off or slow, but then above a certain point it will cross over and run on a higher Max curve.

I might just try running them on my GPU curve for now.
